builtin/logical/pki/path_issue_sign.go

```diff
@@ -192,21 +195,29 @@ func (b *backend) pathIssueSignCert(
 
 	case "pem_bundle":
 		resp.Data["issuing_ca"] = cb.IssuingCA
-		resp.Data["certificate"] = fmt.Sprintf("%s\n%s", cb.Certificate, cb.IssuingCA)
+		resp.Data["issuing_ca_chain"] = cb.IssuingCAChain
+		resp.Data["certificate"] = strings.TrimSpace(fmt.Sprintf("%s\n%s\n%s", cb.Certificate, cb.IssuingCA, cb.IssuingCAChain))
 		if !useCSR {
 			resp.Data["private_key"] = cb.PrivateKey
 			resp.Data["private_key_type"] = cb.PrivateKeyType
-			resp.Data["certificate"] = fmt.Sprintf("%s\n%s\n%s", cb.PrivateKey, cb.Certificate, cb.IssuingCA)
+			resp.Data["certificate"] = strings.TrimSpace(fmt.Sprintf("%s\n%s\n%s\n%s", cb.PrivateKey, cb.Certificate, cb.IssuingCA, cb.IssuingCAChain))
 		}
 
 	case "der":
 		resp.Data["certificate"] = base64.StdEncoding.EncodeToString(parsedBundle.CertificateBytes)
 		resp.Data["issuing_ca"] = base64.StdEncoding.EncodeToString(parsedBundle.IssuingCABytes)
+		for _, cert := range parsedBundle.IssuingCAChainBytes {
+			resp.Data["issuing_ca_chain"] = fmt.Sprintf("%s\n%s", resp.Data["issuing_ca_chain"], base64.StdEncoding.EncodeToString(cert))
+		}
 		if !useCSR {
 			resp.Data["private_key"] = base64.StdEncoding.EncodeToString(parsedBundle.PrivateKeyBytes)
 		}
 	}
 
+	if s, ok := resp.Data["issuing_ca_chain"].(string); ok && len(s) == 0 {
```

Comment:
This was a weird thing after I got a panic in a unit test.  I will have to look into why I did this conversion a little more.  I think your solution might be the right one but I have to dig a little deeper.